docs/index.rst

```diff
@@ -16,6 +16,38 @@ gravitational-wave astronomy, please consider `contributing <https://github.com/
 providing feedback, examples, documentation or helping to improve the core
 library and application suite.
 
+.. carousel::
+    :show_controls:
+    :show_indicators:
+    :show_dark:
+    :show_shadows:
+    :show_captions_below:
+
```

Comment:
Suggest a reordering here. PyCBC's initial purpose was finding GW signals and working with GW data, inference came later. I would prefer to highlight the last two items *first* in the carousel (Preferred order would be [4,3,1,2] compared to current order).

Reply:
Sounds good, reordered as suggested.  The general idea is it should hopefully be easy for people to add more 'highlight' items here and provide a more friendly visual. I didn't really think through the order carefully. It should also autoplay after a bit.